sql/moz-fx-data-shared-prod/fxci/worker_costs_v1/query.sql

```diff
@@ -0,0 +1,9 @@
+-- Query for fxci.worker_costs_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+SELECT
+  *
+FROM
+  table
```

Comment:
Do we need this query?

Reply:
Not to my knowledge. This was auto-generated by `./bqetl query create` and I didn't really understand what it was for but figured it served some kind of purpose.

I can remove if it's not needed.

Reply:
Can I also get rid of the `metadata` files and `view.sql`?